packages/rapier/src/lib/lib/createCollidersFromChildren.ts

```diff
@@ -2,6 +2,15 @@ import { ActiveEvents, Collider, ColliderDesc, World, RigidBody } from '@dimforg
 import { Mesh, Quaternion, Vector3, type Object3D } from 'three'
 import type { AutoCollidersShapes } from '../types/types'
 
+const offset = new Vector3()
```

Comment:
Moved these values up in scope so that they wouldn't be created for each collider